src/code/views/app-view.tsx

```diff
@@ -121,6 +121,11 @@ export class AppView extends Mixer<AppViewProps, AppViewState> {
     }
 
     window.addEventListener("click", this.handleWindowClick, true);
+
+    // Ensure the app has focus so that the top level window delete key handler is invoked
+    // if an element is selected right after and file load and the delete key is pressed.
```

Comment:
```suggestion
    // if an element is selected right after a file load and the delete key is pressed.
```

Reply:
Fixed in https://github.com/concord-consortium/building-models/pull/712/commits/c024f06fcd5faf2e22d805ee1739cf942de70f34